features/bulk-republishing-content.feature

```diff
@@ -35,3 +35,13 @@ Feature: Bulk republishing content
     Given Published organisation "About us" pages exist
     When I request a bulk republishing of all published organisation "About us" pages
     Then I can see all published organisation "About us" pages have been queued for republishing
+
+  Scenario: Republish non-editionable content types
+    Given Contacts exist
+    When I select all of type "Contact" for republishing
+    Then I can see all of type "contact" have been queued for republishing
+
+  Scenario: Republish editionable content types
+    Given Case Studies exist
+    When I select all of type "CaseStudy" for republishing
```

Comment:
Minor, but these `When` step names would be more consistent with others:
- `I request a bulk republishing of all content of type "Contact"`
- `I request a bulk republishing of all content of type "CaseStudy"`

Reply:
Do you mind if I leave it as-is, despite being less consistent here, as I slightly prefer the way the original scans?